scripts/missions/asa/05_Enemy_Of_The_Empire_I.lua

```diff
@@ -0,0 +1,286 @@
+-- A Shantotto Ascension
+-- A Shantotto Ascension M5
+-----------------------------------
+-- !addmission 11 4
+-- Andrause : !pos 22.72 -5.13 18.51 252
+-----------------------------------
+require('scripts/globals/missions')
+require('scripts/settings/main')
+require('scripts/globals/interaction/mission')
+require('scripts/globals/zone')
+-----------------------------------
+local norgID = require("scripts/zones/Norg/IDs")
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.ASA, xi.mission.id.asa.ENEMY_OF_THE_EMPIRE_I)
+
+local mobList =
+{
+    {"Bigclaw", "Bigclaw"},
+    {"Brook Sahagin", "BrookSahagin"},
+    {"Riparian Sahagin", "RiparianSahagin"},
+    {"Rivulet Sahagin", "RivuletSahagin"},
+    {"Royal Leech", "RoyalLeech"},
+    {"Undead Bats", "UndeadBats"},
+    {"Grotto Pugil", "GrottoPugil"},
+    {"Sea Bonze", "SeaBonze"},
+    {"Blubber Eyes", "BlubberEyes"},
+    {"Bog Sahagin", "BogSahagin"},
+    {"Marsh Sahagin", "MarshSahagin"},
+    {"Rock Crab", "RockCrab"},
+    {"Razorjaw Pugil", "RazorjawPugil"},
+    {"Sahagin Parasite", "SahaginParasitd"},
+    {"Swamp Sahagin", "SwampSahagin"},
+    {"Devil Manta", "DevilManta"},
+    {"Dire Bat", "DireBat"},
+    {"Lagoon Sahagin", "LagoonSahagin"},
+    {"Delta Sahagin", "DeltaSahagin"},
+    {"Coastal Sahagin", "CoastalSahagin"},
+    {"Mousse", "Mousse"},
+    {"Robber Crab", "RobberCrab"},
+    {"Shore Sahagin", "ShoreSahagin"},
+}
+
+local function handleTradeEvent(player, trade)
+    local mobOne = mission:getVar(player, 'MobOne')
+
+    -- Trading Soul Plates
+    if mobOne ~= 0 and npcUtil.tradeHasExactly(trade, xi.items.SOUL_PLATE) then
+        local mobTwo = mission:getVar(player, 'MobTwo')
+        local mobThree = mission:getVar(player, 'MobThree')
+        local platesTraded = mission:getVar(player, 'Plates')
+        local item = trade:getItem(0)
+        local plateData = item:getSoulPlateData()
+
+        -- Check if traded Soulplate is one of the requested mobs
+        for value, data in pairs(mobList) do
+            if data[2] == plateData.name then
+                mission:setVar(player, 'Found', value)
+                mission:setVar(player, 'Plates', platesTraded + 1)
+
+                -- Assign 99 to traded mob's var to mark it as completed
+                if value == mobOne then
+                    mission:setVar(player, 'MobOne', 99)
+                elseif value == mobTwo then
+                    mission:setVar(player, 'MobTwo', 99)
+                elseif value == mobThree then
+                    mission:setVar(player, 'MobThree', 99)
+                end
+
+                player:confirmTrade()
+                return mission:progressEvent(241)
+            end
+        end
+
+        -- Wrong soulplate traded
+        return mission:progressEvent(242)
+    end
+end
+
+mission.reward =
+{
+    nextMission = { xi.mission.log_id.ASA, xi.mission.id.asa.ENEMY_OF_THE_EMPIRE_II },
+    ki = xi.ki.BLACK_BOOK,
```

Comment:
Just to be consistent, please have nextMission as the last item in the reward block